backend/app/api/src/helpers/SetupStepsUpdater.ts

```diff
@@ -0,0 +1,194 @@
+import {
+    Organization,
+    OrganizationRegistrationPeriod,
+    Platform,
+} from "@stamhoofd/models";
+import { SetupStepType, SetupSteps } from "@stamhoofd/structures";
+
+export class SetupStepUpdater {
+    private static readonly STEP_TYPE_OPERATIONS: Record<SetupStepType, (setupSteps: SetupSteps,
+        organization: Organization,
+        platform: Platform) => void> = {
```

Comment:
Probeer dit wat leesbaarder te formateren. Bv. door bovenaan
`type SetupStepOperation = (setupSteps: SetupSteps, organization: Organization, platform: Platform) => void;` te zetten.

```suggestion
    private static readonly STEP_TYPE_OPERATIONS: Record<SetupStepType, SetupStepOperation> = {
```